cli/src/utils/cli-flags.ts

```diff
@@ -29,7 +29,13 @@ export default {
         return val;
       }
       error(`Expected a formatted duration or ISO 8601 string format but received: ${val}`);
-      throw new Error('Should not get here');
+    },
+  }),
+  keyValue: flags.build({
+    parse(val) {
+      const kvMatch = val.match(/^([^=]+)=(.*)$/);
```

Comment:
Can you add a test for the edge case of `VARIABLE=` empty string? I guess the only way to pass multiline strings as variables is to put them in an environment variable first?

Reply:
Adding an empty variable to the test.

Multiline variables are easy using The Power of Bash (the `$'` is just to escape the single quotes, nothing to do with the multiline string).  But actually trying it revealed a bug in the `keyValue` arg type, so pushing a fix.
```sh
flutterby:counter-class ariels$ node  ~/Dev/reshuffle/cli/bin/run env --set $'a=
Humpty Dumpty sat on a wall,
Humpty Dumpty had a great fall.
All the king\'s horses
And all the king\'s men
Couldn\'t put Humpty together again.'
flutterby:counter-class ariels$ node  ~/Dev/reshuffle/cli/bin/run env --get a
a=
Humpty Dumpty sat on a wall,
Humpty Dumpty had a great fall.
All the king's horses
And all the king's men
Couldn't put Humpty together again.
```